hangman.py

```diff
@@ -42,12 +35,7 @@ def choose_word(wordlist):
     """
     return random.choice(wordlist)
 
-# end of helper code
-
-# -----------------------------------
 
-# Load the list of words into the variable wordlist
-# so that it can be accessed from anywhere in the program
```

Comment:
I wouldn't delete that comment - it has a description for that veriable

Reply:
Done